os/binfmt/binfmt_loadbinary.c

```diff
@@ -165,7 +166,7 @@ int load_binary(int binary_idx, FAR const char *filename, load_attr_t *load_attr
 		ret = load_module(bin);
 		if (ret < 0) {
 			errcode = ret;
-			berr("ERROR: Failed to load program '%s': %d\n", filename, errcode);
+			berr("%s load '%s': %d\n",clog_message_str[CMN_LOG_FAILED_OP], filename, errcode);
```

Comment:
check reconstructed message -> operation file load binfmt_loadmemory errorcode --> is this making sense ?